examples/ruby/create_themes.rb

```diff
@@ -0,0 +1,53 @@
+microsoft_like = {:name=>"microsoft", :settings=>{:background_color=>"#ffffff", :base_font_size=>"12px", :color_collection_id=>"80e60a97-c02b-4a41-aa05-83522ee2144b", :font_color=>"#3a5998", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#FFB900", :primary_button_color=>"#00A4EF", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"#7FBA00", :tile_background_color=>"#ffffff", :tile_text_color=>"#F25022", :title_color=>"#737373", :warn_button_color=>"#3a5998"}}
+facebook_like = {:name=>"facebook", :settings=>{:background_color=>"#f6f6f6", :base_font_size=>"12px", :color_collection_id=>"", :font_color=>"#3a5998", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#3a5998", :primary_button_color=>"#3a5998", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#f6f6f6", :tile_text_color=>"#8b9dc3", :title_color=>"#4385f5", :warn_button_color=>"#3a5998"}}
+datadog_like = {:name=>"datadog", :settings=>{:background_color=>"#fdf0ff", :base_font_size=>"12px", :color_collection_id=>"5b121cce-cf79-457c-a52a-9162dc174766", :font_color=>"#774aa4", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#774aa4", :primary_button_color=>"#774aa4", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#fdf0ff", :tile_text_color=>"#774aa4", :title_color=>"#774aa4", :warn_button_color=>"#774aa4"}}
+strava_like = {:name=>"strava", :settings=>{:background_color=>"#fff8f5", :base_font_size=>"12px", :color_collection_id=>"7c79334a-9912-4ca1-be6a-35756782ae09", :font_color=>"#FC4C02", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#FC4C02", :primary_button_color=>"#FC4C02", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#fff8f5", :tile_text_color=>"#FC4C02", :title_color=>"#FC4C02", :warn_button_color=>"#FC4C02"}}
+sendgrid_like =  {:name=>"sendgrid", :settings=>{:background_color=>"#f6f6f6", :base_font_size=>"12px", :color_collection_id=>"1bc1f1d8-7461-4bfd-8c3b-424b924287b5", :font_color=>"#1A82e2", :font_family=>"Arial", :font_source=>"", :info_button_color=>"#294661", :primary_button_color=>"#294661", :show_filters_bar=>true, :show_title=>true, :text_tile_text_color=>"", :tile_background_color=>"#f6f6f6", :tile_text_color=>"#1A82e2", :title_color=>"#1A82e2", :warn_button_color=>"#294661"}}
+​
+all_themes = [airbnb_like, gcp_like, aws_like, microsoft_like, facebook_like, datadog_like, strava_like, sendgrid_like]
+​
+# pull the list of themes we created above and create them all in our instance
+all_themes.each { | theme|
+	sdk.create_theme(theme)
+}
```

Comment:
extra space in `|  theme|` - I think we generally don't pad these

Also, (and I had to look this up) we stick to [this convention](https://stackoverflow.com/questions/5587264/do-end-vs-curly-braces-for-blocks-in-ruby)

so you should either
```ruby
all_themes.each { |theme| sdk.create_theme(theme) }
```
or
```ruby
all_themes.each do |theme|
  sdk.create_theme(theme)
end
```